.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -64,6 +61,15 @@ body:
     validations:
       required: false
 
+  # Note that the user should always upload the report rather than pasting it.
+  - type: markdown
```

Comment:
I'm not sure if it's because of the wording, but the tip seems to have made things worse.

Reply:
Maybe I should rethink it.

Reply:
Perhaps it's the word `pasting`, should we try `Please **upload** the log file by clicking "Attach files"`.?

Reply:
I think that's better probably, @0x5bfa what do you think?

Reply:
I don't have preference

Reply:
It would be interesting to see if there's a bot that could automate  this by detecting a log in text and wrapping it with a `<details/>` block.